### PR TITLE
App is now adding users to the database. See at http://localhost:3003…

### PR DESCRIPTION
**Pull request by vkanicka**
…/users/

Added to app.js: require("dotenv").config(); 

Created sessions.js controller as seen in thiago's fruits w/ db user sessions lesson 3/13/21

Still troubleshooting login fetch error / no changes to login error yet.

User sign up now working - database reflects user objects.

### CODE REVIEW

File: Controllers/sessions.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const router = express.Router();
+const bcrypt = require('bcrypt');
+const User = require('../Models/userModel')
+
+router.get('/sessions', (req, res) => {
```
Comment (am852410) [inline Controllers/sessions.js:6]: Just curious what this is doing?

File: Controllers/sessions.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  console.log(`session get router`)
+})
+
+// USER LOGIN ROUTE (CREATE SESSION ROUTE)
+router.post("/sessions", (req, res) => {
```
Comment (am852410) [inline Controllers/sessions.js:11]: Wasn't there already a log in route?

File: server.js [hunk 3]
```diff
@@ -1,52 +1,163 @@
...
+//--------------------------------------------
+//SESSION
 const session = require("express-session");
+//--------------------------------------------
 
-app.use(express.json());
+//--------------------------------------------
+//MONGO
+const mongoURI = process.env.MONGODBURI
+const MongoDBStore = require('connect-mongodb-session')(session);
```
Comment (am852410) [inline server.js:40]: What does this do? Just curious

File: Controllers/sessions.js (hunk 1 above)
Comment (vkanicka) [reply Controllers/sessions.js:6]: I took that out in the last commit

File: Controllers/sessions.js (hunk 2 above)
Comment (vkanicka) [reply Controllers/sessions.js:11]: I deleted sessions.js in the last commit

File: Controllers/sessions.js (hunk 1 above)
Comment (am852410) [reply Controllers/sessions.js:6]: Did you do it on this PR because it shows that it is still here. If you deleted it would not exist in the PR

Comment (vkanicka) [reply Controllers/sessions.js:6]: There are two PRs

**Pull request merged**